tests/scripts/run_moreh_microbenchmark.sh

```diff
@@ -32,9 +32,9 @@ run_profiling_test() {
   pytest --capture=tee-sys $TT_METAL_HOME/tests/scripts/test_moreh_microbenchmark.py::test_matmul_dram -k $ARCH_NAME
   pytest --capture=tee-sys $TT_METAL_HOME/tests/scripts/test_moreh_microbenchmark.py::test_matmul_l1 -k $ARCH_NAME
 
-  if [[ "$ARCH_NAME" == "wormhole_b0" ]]; then
+  if [[ "$ARCH_NAME" != "grayskull" ]]; then
   pytest --capture=tee-sys $TT_METAL_HOME/tests/scripts/test_moreh_microbenchmark.py::test_matmul_single_core_sharded -k $ARCH_NAME
-  pytest --capture=tee-sys $TT_METAL_HOME/tests/scripts/test_moreh_microbenchmark.py::test_dram_read_12_core -k $ARCH_NAME
```

Comment:
@yugaoTT I tried running `test_matmul_single_core_sharded` on WH and I was seeing segfaults

Reply:
can you try run run_moreh_microbenchmark.sh first? It might be the output csv is not created yet

Reply:
also build with ./build_metal.sh and make sure the c++ test are compiled